airbyte-workers/src/test/java/io/airbyte/workers/general/DefaultReplicationWorkerTest.java

```diff
@@ -441,7 +441,8 @@ void testPopulatesOutputOnSuccess() throws WorkerException {
     when(messageTracker.getDestinationOutputState()).thenReturn(Optional.of(new State().withState(expectedState)));
     when(messageTracker.getTotalRecordsEmitted()).thenReturn(12L);
     when(messageTracker.getTotalBytesEmitted()).thenReturn(100L);
-    when(messageTracker.getTotalStateMessagesEmitted()).thenReturn(3L);
+    when(messageTracker.getTotalSourceStateMessagesEmitted()).thenReturn(3L);
+    when(messageTracker.getTotalDestinationStateMessagesEmitted()).thenReturn(3L);
```

Comment:
I would use a different value to make sure we do not confuse the two fields. (bad copy paste typically)